src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
```

Comment:
Great work using `map` here for each button group but we can go further and use `map` on the button groups themselves:

```javascript
const btnGroups = [btnGroup1, btnGroup2, btnGroup3, btnGroup4, btnGroup5];
```
For which we can do the ff:

```javascript
return (
  <div>
   btnGroups.map(btnGroup => btnGroup.map(
     // here you will do the exact same map you did in each Button
   ))
 </div>
)
```